controller/src/reports/utils.py

```diff
@@ -277,6 +277,8 @@ def get_subscription_stats_for_cycle(subscription, start_date=None):
 
     Determine the cycle by the provided start_date.
     """
+    import ipdb; ipdb.set_trace()
```

Comment:
Need to remove ipdb debugger from the code, otherwise, everything else looks great!